src/main/java/com/vacation_bot/domain/models/VacationTotalModel.java

```diff
@@ -0,0 +1,50 @@
+package com.vacation_bot.domain.models;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.CompoundIndex;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@Document
+@CompoundIndex(name = "unique_idx", def = "{ 'user-id': 1, 'year': 1 }", unique = true)
+public class VacationTotalModel {
```

Comment:
You have to create a new repository for this document. It's ok to add it with the next PR.